frontend/src/App.tsx

```diff
@@ -16,9 +19,11 @@ export const App: React.FC = () => <>
             </Switch>
         </Root>
     </Router>
-</>;
+</RelayEnvironmentProvider>;
 
 
 const RealmPage: React.FC<RouteComponentProps<{ path?: string }>> = ({ match }) => (
-    <Realm path={match.params.path?.split("/") ?? []} />
+    <Suspense fallback="TODO">
```

Comment:
```suggestion
    <Suspense fallback="Loading! (TODO)">
```

I'm fine with merging this without a proper loading animation, but yeah that change pls.

Reply:
Done.